dev/devicelab/test/runner_test.dart

```diff
@@ -40,5 +41,31 @@ void main() {
       expect(printLog[0], 'Consistently failed across all 3 executions.');
       expect(printLog[1], 'flaky: false');
     });
+
+    test('Regression test for https://github.com/flutter/flutter/issues/155475.', () async {
```

Comment:
nit: I think usually we don't name tests after the GitHub issue, but instead what the test is actually checking for.  Doesn't matter that much though.

Reply:
Done!